src/pages/apply.js

```diff
@@ -0,0 +1,1169 @@
+          <StyledSelect
+            options={genderOptions}
+            onChange={handleSelectChange}
+            onBlur={e => handleTouched(e, "gender")}
+            required
+          />
+          <ErrorDiv>
+            {formState.gender.isTouched && !formState.gender.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Country <Required>*</Required>
+          </span>
+          <div>
+            <small>
+              Please provide the current country you’re located in or spend the
+              most time if it’s not your country of origin
+            </small>
+          </div>
+          <StyledSelect
+            options={countryOptions}
+            onChange={handleSelectChange}
+            onBlur={e => handleTouched(e, "country")}
+            required
+          />
+          <ErrorDiv>
+            {formState.country.isTouched && !formState.country.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Time Zone <Required>*</Required>
+          </span>
+          <div>
+            <small>
+              Please choose your current time zone to help us schedule calls.
+            </small>
+          </div>
+          <StyledSelect
+            options={timezoneOptions}
+            onChange={handleSelectChange}
+            onBlur={e => handleTouched(e, "timezone")}
+            required
+          />
+          <ErrorDiv>
+            {formState.timezone.isTouched && !formState.timezone.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>Social network(s)</span>
+          <div>
+            <small>
+              Please indicate your preferred social network(s) handle(s)
+            </small>
+          </div>
+          <Input
+            type="text"
+            name="socialNetworks"
+            value={formState.socialNetworks.value}
+            onChange={handleInputChange}
+            maxLength="255"
+            onBlur={handleTouched}
+          />
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            What is your current occupation? <Required>*</Required>
+          </span>
+          <Input
+            type="text"
+            name="title"
+            value={formState.title.value}
+            onChange={handleInputChange}
+            maxLength="255"
+            onBlur={handleTouched}
+            required
+          />
+          <ErrorDiv>
+            {formState.title.isTouched && !formState.title.isValid && (
+              <RequiredError />
+            )}
+          </ErrorDiv>
+        </StyledLabel>
+        <StyledLabel>
+          <span>
+            Are you affiliated with any organization(s)? <Required>*</Required>
+          </span>
+          <StyledSelect
```

Comment:
This yes/no question should definitely not be a search select 😄 - maybe radio buttons?